instrumentation/jdbc/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/jdbc/datasource/DataSourceInstrumenters.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.jdbc.datasource;
+
+import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.instrumentation.api.instrumenter.Instrumenter;
+import io.opentelemetry.instrumentation.api.instrumenter.SpanNameExtractor;
+import io.opentelemetry.instrumentation.api.instrumenter.code.CodeAttributesExtractor;
+import io.opentelemetry.instrumentation.api.instrumenter.code.CodeSpanNameExtractor;
+import javax.sql.DataSource;
+
+public final class DataSourceInstrumenters {
+  private static final String INSTRUMENTATION_NAME = "io.opentelemetry.javaagent.jdbc";
+
+  private static final Instrumenter<DataSource, Void> INSTRUMENTER;
```

Comment:
would `DataSource` -> `Method` make sense?

Reply:
I tried that first - but I got different class names in tests (`method.getDeclaringClass()` vs `dataSource.getClass()`). I thought that it's probably a bit better to use the actual data source type in the attribute value, not the abstract/base class type.

Reply:
makes sense 👍